netnode/test_netnode.py

```diff
@@ -7,10 +7,10 @@
 import idaapi
 import pytest
 
-import netnode
+from . import netnode
```

Comment:
i'm not sure the most correct construct here. (and in the other import statement change.)

i think using absolute importing is probably best, when feasible.